dll/cpl/joy/joy.c

```diff
@@ -113,11 +122,40 @@ AdvancedPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 INT_PTR CALLBACK
 CustomPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
+    WCHAR szBuf[2];
+    HWND hComboHwnd;
+
     UNREFERENCED_PARAMETER(lParam);
 
     switch (uMsg)
     {
         case WM_INITDIALOG:
+            szBuf[1] = UNICODE_NULL;
```

Comment:
```suggestion
        {
            HWND hComboHwnd;
            WCHAR szBuf[2] = {0, 0};
```

Reply:
I prefer to keep UNICODE_NULL but I take the other part